src/isaw.policy/src/isaw/policy/metadata.py

```diff
@@ -71,24 +99,39 @@ def update(self):
                 if tag:
                     self.metatags.append((u'article:tag', tag.decode('utf8')))
 
-    @property
+    @memoizedproperty
```

Comment:
This memoizes per instance, and re-evaluates if the image is updated?

Reply:
It's pretty dumb, it just memoizes on the instance of the view class. It just ensures that the `image_url` property is only computed once even though it's used multiple time within `update`.